TAF/utils/scripts/docker/docker-compose-end-to-end.yaml

```diff
@@ -0,0 +1,63 @@
+  app-service-http-export:
+    image: ${appService}
+    ports:
+      - "48096:48096"
+    entrypoint: ["/app-service-configurable"]
+    command: ["--registry","--confdir=/res", "--skipVersionCheck=true", "--overwrite"]
+    container_name: app-service-http-export
+    hostname: app-service-http-export
+    environment:
+      edgex_registry: consul://edgex-core-consul:8500
+      edgex_service: http://app-service-http-export:48096
+      edgex_profile: http-export
+      Service_Host: app-service-http-export
+      Clients_CoreData_Host: edgex-core-data
+      Clients_Logging_Host: edgex-support-logging
+      MessageBus_SubscribeHost_Host: edgex-core-data
+      Database_Host: edgex-mongo
+      Database_Username: appservice
+      Database_Password: password
+      EDGEX_SECURITY_SECRET_STORE: "${SECURITY_ENABLED}"
+      Writable_Pipeline_Functions_HTTPPostJSON_Parameters_url: http://172.17.0.1:8080
```

Comment:
Will this IP address always be `172.17.0.1`?

Reply:
@lenny-intel As far as I understand, if using default bridge network on docker, we can always connect to docker host by 172.17.0.1 from container.

Reply:
please see whether `host.docker.internal` can replace with the hard code ip address
https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds

Reply:
don't use 8080 port because it is easy to be conflicted

Reply:
@cloudxxx8  Update the http server port to 7770.
But using `host.docker.internal` to instead `172.17.0.1` occurred the following error when posting data.
`level=ERROR ts=2020-04-15T08:09:25.239280982Z app=AppService-http-export source=runtime.go:181 error="Post http://gateway.docker.internal:7770: dial tcp: lookup gateway.docker.internal on 127.0.0.11:53: no such host" correlation-id=6814a721-ec86-4c7b-8eb0-ba9d5aea1964 msg="Pipeline function #2 resulted in error"`

Reply:
I tried the IP `172.17.0.1` on sandbox, it worked on both env., x86_64 and arm64.

Reply:
@lenny-intel it is the default ip address of the host for docker, and the http server is running on the host.  We made it become configurable in the environment variable.  Is is ok with you?

Reply:
Yes. Perfect! THX